### PR TITLE
chore: merge `release/v1.26.4` branch into the `releases` branch

### PR DESCRIPTION
**Pull request by rjan90**
## Proposed Changes
Lifting a couple of bugfixes into a patch release that was missed in the v1.26.x branches, as point out by [Riba here](https://filecoinproject.slack.com/archives/CP50PPW2X/p1714628806367119?thread_ts=1713976109.342789&cid=CP50PPW2X).

Most notebly the ListClaimsCmd has been refined to correctly filter out expired claims, the SectorOnChainInfo in lotus/chain/actors/builtin/miner now correctly points to v13 GST. As well as smaller updates to error messages and parts missed in the release engineering process.

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: documentation/en/api-v0-methods.md [hunk 1]
```diff
@@ -5965,12 +5965,12 @@ Response:
...
     "ExpectedDayReward": "0",
     "ExpectedStoragePledge": "0",
-    "ReplacedSectorAge": 10101,
+    "PowerBaseEpoch": 10101,
     "ReplacedDayReward": "0",
     "SectorKeyCID": {
       "/": "bafy2bzacea3wsdh6y3a36tb3skempjoxqpuyompjbmfeyf34fi3uy6uue42v4"
     },
-    "SimpleQAPower": true
+    "Flags": 0
```
Comment (aarshkshah1992) [inline documentation/en/api-v0-methods.md:5973]: What change is causing this diff ?

**Review by aarshkshah1992 — APPROVED** [review]
Just one question

File: documentation/en/api-v0-methods.md (hunk 1 above)
Comment (aarshkshah1992) [reply documentation/en/api-v0-methods.md:5973]: This is because we have moved to miner13 types. lgtm.

Comment (rjan90) [reply documentation/en/api-v0-methods.md:5973]: `make gen` generated those after we changed to miner13 in https://github.com/filecoin-project/lotus/pull/11982/commits/941d0befe3b30b910291738e623579ea852c28d9

**Review by ZenGround0 — APPROVED** [review]

**Comment by jennijuju** [comment]
shouldnt this be merged before the next v1.27.x release?

**Comment by rjan90** [comment]
> shouldnt this be merged before the next v1.27.x release?

No, I will close this one. As this newer got

**Pull request closed without merging**